app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    sum_raiting = 0
```

Comment:
simplify to:
return sum(player.get_rating() for player in players)

Reply:
fixed